app/views/layouts/beta_layout.slim

```diff
@@ -56,9 +56,23 @@ html lang="en"
                           div.nav-bar-icon-container
                             div.nav-bar-icon
                       ul.dropdown-menu id="mobileNavDropdown" style="display: none"
-                        -if @user.member?
-                          li.dropdown-header.filter-item Essentials
-                          ==render partial: "custom_links"
+                        -custom_links=render('custom_links')
+                        -if @user.member? || custom_links.length > 0
+                          li.dropdown-header.filter-item Quick Links
+                          -if custom_links.length > 0
```

Comment:
Line 62 not needed since you aren't doing a divider anymore and you already checked length.

Reply:
It could be a signed in user on site without custom links. We still would want the Quick Links section to render, just not the subsection within it that reveals custom_links.